### ISSUE TITLE
Look for fitting sound effects from vanilla to add to your Events

### ISSUE THREAD

**Issue by Elimdur** [MoreEventsMod/More_Events_Mod_Beta#259]
Many of our events - includin mine - are mostly silent.

We should go through our files and add fitting soundeffects, when we have the time.

I will go through my events first, then go on with the already released old events. Every active coder should go through his events as well.

**Comment by Tuttu** [comment]
Good idea. I will do that with my own events as well as soon as I'm done with the Lunar Gate event.
I will also check if I can upload them on my website for easy reference. :)

**Comment by Elimdur** [comment]
That would be a long list

**Comment by Elimdur** [comment]
DONE :
- Ancient Robots
- Sleepers
- Extinct Abductors
- Demon Ship

**Comment by tzerrell** [comment]
This is a good idea. I used to try to write soundfx into my events but eventually stopped (probably mostly because I typically play with sound off). But I'd like to get them in my events again.
